### PR TITLE
 create delete event function  #81

### PR DESCRIPTION
**Pull request by aboodswiti**
Delete Event 
BackEnd & FrontEnd

### CODE REVIEW

File: server/database/queries/eventDelete.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const dbconniction = require("../db_connection");
+
+const eventDelete = (eventId) => {
+  const sql = {
+    text: "DELETE FROM events WHERE event_id = $1 ",
+    values: [eventId],
+  };
+  return dbconniction.query(sql).catch((err) => console.log(err));
```
Comment (yaqootturman) [inline server/database/queries/eventDelete.js]: We talked about this, in queries should not use promise.
you can write return dbconniction.query(sql) and that will be enough

File: server/controllers/deleteEvent.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const eventDelete = require("../database/queries/eventDelete");
+
+const deleteEvent = (req, res) => {
+  const eventId = req.body.eventId;
+
+  eventDelete(eventId);
+  res.send();
```
Comment (yaqootturman) [inline server/controllers/deleteEvent.js]: So here you send a response for nothing, that means even error will be response

File: client/src/components/Layout/UserHome/UserHome.js [hunk 3]
```diff
@@ -3,24 +3,32 @@ import Profile from "../../../Images/Ellipse 2.jpg";
...
+    eventId: "",
+    eventName:""
   };
   componentDidMount() {
+
     const id = 3;
     axios
       .get(`/api/user-events/${id}`)
       .then((res) => this.setState({ events: res.data }))
       .catch((err) => console.log(err));
```
Comment (yaqootturman) [inline client/src/components/Layout/UserHome/UserHome.js]: Handle the error based on controller response

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: server/controllers/index.js [hunk 4]
```diff
@@ -15,5 +16,6 @@ router.get('/api/events', getAllEvents)
 router.get('/api/available-rooms', searchAvailableRooms)
 router.post('/api/login', login)
 router.post('/api/booking', addEvent)
+router.post('/api/delete-event', deleteEvent)
```
Comment (yaqootturman) [inline server/controllers/index.js:26]: use router.delete

Comment (yaqootturman) [reply server/controllers/index.js:26]: and same for axios

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by tasneembhiri — APPROVED** [review]

**Review by bayanseder — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Pull request merged**